mlflow/R/mlflow/DESCRIPTION

```diff
@@ -40,17 +40,20 @@ Imports:
     xml2,
     yaml,
     zeallot
-Suggests: 
+Suggests:
   covr,
   carrier,
+  dplyr,
   keras,
   lintr,
+  sparklyr,
   testthat (>= 2.0.0),
   stringi (< 1.4.4),
   xgboost,
-  h2o
+  h2o,
+  mleap
```

Comment:
seems like description is tidy (before you added mlleap). Suggest to run `usethis::use_tidy_description()`

Reply:
I think it would be great at some point to add pre-commit.com hooks to this repo, then you could use https://lorenzwalthert.github.io/precommit/articles/available-hooks.html#deps-in-desc for this.

Reply:
I suggested to use pre-commit for MLflow: https://github.com/mlflow/mlflow/issues/3823